README.md

```diff
@@ -62,6 +63,9 @@ Video](https://img.youtube.com/vi/-Xd84hlIjkI/0.jpg)](https://www.youtube.com/wa
 
 - `FLASK_STATIC_DIGEST_HOST_URL` has been added to configure an optional external host, aka. CDN ([explained here](#configuring-this-extension))
 - If your blueprints have static files they will get digested now too (including nested blueprints!)
+- Brotli support has been added
+  - `FLASK_STATIC_DIGEST_GZIP_FILES` is depreciated and will be removed in 0.5
```

Comment:
I think we can remove this from the readme. Folks who already have it defined will see the deprecation notice and anyone new to the library can use `FLASK_STATIC_COMPRESSION` from the get go.